### PR TITLE
Add github issue template

### PR DESCRIPTION
**Pull request by ralphtheninja**
Closes https://github.com/deltachat/deltachat-desktop/issues/307

### CODE REVIEW

**Comment by ralphtheninja** [comment]
Stole it from `deltachat-core` and tweaked it a bit.

**Comment by ralphtheninja** [comment]
Rebasing on latest master.

**Comment by hpk42** [comment]
FWIW i think the tempalte is a bit too verbose ... after mentioning the version/platform stuff maybe just some suggestions on what to add ina  few "-" items?

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<!--
+Please fill in as much as you can below (leaving out if not applicable is ok). This is a bug report forum, if you like to discuss a new feature or submit suggestions, please use the community forum at https://support.delta.chat
```
Comment (hpk42) [inline .github/ISSUE_TEMPLATE.md]:
```suggestion
Please fill in as much as you can below (leaving out if not applicable is ok). This is a bug report and feature-tracking forum, if you like to discuss a new feature, please use the community forum at https://support.delta.chat. 
```

**Comment by hpk42** [comment]
much better, thanks! IMO fine to merge (preferably with my little suggestion applied)

**Pull request merged**